### PR TITLE
test: fix test

### PR DESCRIPTION
**Pull request by 0OPT**
* fix Issue Test #4
* fix bug introduced in ff512348710284fb1c160da059d8f1b63217397d

### CODE REVIEW

**Review by TMP-0 — APPROVED** [review]
Fine

**Review by TMP-0 — APPROVED** [review]
Fine

**Review by TMP-0 — COMMENTED** [review]
Comment

**Review by TMP-0 — CHANGES_REQUESTED** [review]
Test

File: main.go [hunk 1]
```diff
@@ -4,5 +4,7 @@ import "fmt"
 
 func main() {
 	tmp := 12
+	tmp1 := "Test"
 	fmt.Printf("Hello World.%d\n", tmp)
+	fmt.Println(tmp1)
```
Comment (TMP-0) [inline main.go:9]: Need improvement.

Comment (0OPT) [reply main.go:9]: Fine.

Comment (TMP-0) [inline main.go:9]: Another Test.

**Review by TMP-0 — APPROVED** [review]
Final Test.

**Pull request merged**